### PR TITLE
use full header on comment history

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Puts the whole header on the user comment history

## What is the value of this and can you measure success?
makes comments feel more like a part of the full website instead of a side page. Also fixes the page rendering wonkily on phones

## Screenshots (before/after)

screen shot 2018-01-15 at 07 55 56
screen shot 2018-01-15 at 07 55 54

### CODE REVIEW

File: common/app/model/meta.scala [hunk 1]
```diff
@@ -257,7 +257,7 @@ final case class MetaData (
 
   val hasSlimHeader: Boolean =
     contentWithSlimHeader ||
-      sectionId == "identity" ||
+      (sectionId == "identity" && !id.startsWith("/user/")) ||
```
Comment (NataliaLKB) [inline common/app/model/meta.scala]: Oh gosh. It isn't a nice way is it? Not sure how else we can do this though :woman_shrugging:

**Review by NataliaLKB — APPROVED** [review]

File: common/app/model/meta.scala (hunk 1 above)
Comment (walaura) [reply common/app/model/meta.scala]: we could remove the whole line really 😅 The rest of identity pages should be using `identityHeader` anyway.

Comment (NataliaLKB) [reply common/app/model/meta.scala]: You do what you think is best :). I think you know better than me what the identity pages need!

Comment (walaura) [reply common/app/model/meta.scala]: I think every other significant page is ported over but I wanna be more sure (so, wait s bit) before making the change!

**Pull request merged**